typescript-sdk/src/observability/tracer.ts

```
@@ -0,0 +1,229 @@
+
+            return target.startActiveSpan(name, cb);
+          };
+
+        case "startSpan":
+          return (name: string, options?: SpanOptions, context?: Context) =>
+            createLangWatchSpan(target.startSpan(name, options, context));
+
+        default:
+          const value = (target as any)[prop];
+
+          return typeof value === "function" ? value.bind(target) : value;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)